recipes/cuquantum/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = "0.1.0" %}
+{% set patch_version = "30" %}
+
+package:
+  name: cuquantum
+  version: {{ version }}.{{ patch_version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuquantum/{{ version }}/local_installers/cuquantum-linux-x86_64-{{ version }}.{{ patch_version }}-archive.tar.xz   # [linux64]
+  url: https://developer.download.nvidia.com/compute/cuquantum/{{ version }}/local_installers/cuquantum-linux-sbsa-{{ version }}.{{ patch_version }}-archive.tar.xz     # [aarch64]
+  url: https://developer.download.nvidia.com/compute/cuquantum/{{ version }}/local_installers/cuquantum-linux-ppc64le-{{ version }}.{{ patch_version }}-archive.tar.xz  # [ppc64le]
+
+  sha256: 8ad8e98f14275ffe0de02574be5c86224af1c657c41baf02c16440301ffe0aae  # [linux64]
+  sha256: 3dd04cf08f1323318e0e63a7e28bc904c426ced3367aca809fba7e7beef94063  # [aarch64]
+  sha256: 0eb84eef619a1cfab5870fb585200a9869a197866685252e4ca8187322809554  # [ppc64le]
+
+build:
+  number: 2
+  skip: True  # [win or ppc64le or cuda_compiler_version != "11.2"]
+  script_env:
+    # for some reason /usr/local/cuda is not added to $PATH in ppc64le's docker image
+    - CUDA_HOME  # [ppc64le or aarch64]
+  script:
+    - mkdir -p $PREFIX/include                                            # [linux]
+    - cp -r include/* $PREFIX/include/                                    # [linux]
+    - mkdir -p $PREFIX/lib                                                # [linux]
+    - mv lib/*.so* $PREFIX/lib/                                           # [linux]
+
```

Comment:
Is this essentially repackaging?

Reply:
Yes, this is standard practice used in all CUDA packages so that we keep the binary and their (relative) symlinks.